src/utils/user.js

```diff
@@ -0,0 +1,20 @@
+import {get} from 'lodash';
+/**
+ *
+ * @param  {Object} user [description]
+ * @return {String}      [description]
+ */
+export const getNickname = (user) => {
+  return user && user.nickname ? user.nickname : '';
+};
+
+/**
+ * [getAuthorName description]
+ * @param  {Object} user [description]
+ * @return {String}      [description]
+ */
+export const getAuthorName = (user) => {
+  return get(user, 'nickname') || get(user, 'displayName');
```

Comment:
The problem is, the user object here is obtained from the node.js frontend backend helsinki-passport app, which does not contain the nickname field.

If we add a nickname field, this will be obtained directly from the backend API. Is it possible to

a) get the author nickname and other properties directly from :https://api.hel.fi/kerrokantasi-test/v1/users/ endpoint or

b) plug the extra data from the API somehow into the passport user object?

Reply:
If I'm understanding this correctly, both a & b should be possible. Where does one get the nickname at this point?

Reply:
Ah. My bad. Just added the nickname field to the backend master branch, so it should be available in the `users` endpoint `nickname` field.